ml-proto/Makefile

```diff
@@ -21,7 +20,7 @@ OCB =		ocamlbuild $(OCB_FLAGS)
 .PHONY:		all opt unopt
 unopt:		$(NAME_UNOPT)
 opt:		$(NAME_OPT)
-all:		opt unopt test
+all:		winmake.bat opt unopt test
```

Comment:
And it shouldn't! Any patch needs to regenerate the batch file as necessary, regardless of dev platform.